papermill/abs.py

```diff
@@ -32,13 +34,24 @@ def _split_url(self, url):
 
     def read(self, url):
         params = self._split_url(url)
+        output_stream = io.BytesIO()
         block_blob_service = self._block_blob_service(
             account_name=params["account"], sas_token=params["sas_token"]
         )
-        return block_blob_service.get_blob_to_text(
-            container_name=params["container"], blob_name=params["blob"]
+
+        block_blob_service.get_blob_to_stream(
+            container_name=params["container"],
+            blob_name=params["blob"],
+            stream=output_stream,
         )
 
+        output_stream.seek(0)
+        content = output_stream.readlines()
+        return [
+            line.decode("utf-8") if isinstance(line, six.binary_type) else line
+            for line in content
```

Comment:
```suggestion
            for line in output_stream
```
does this work in Python 2 and 3?

Reply:
I like your suggestions. I will change this accordingly.

Reply:
By the way. This works for Python 2 and 3